internal/exec/subscribe.go

```diff
@@ -139,8 +139,12 @@ func (r *Request) Subscribe(ctx context.Context, s *resolvable.Schema, op *query
 				}
 				var out bytes.Buffer
 				func() {
-					// TODO: configurable timeout
-					subCtx, cancel := context.WithTimeout(ctx, time.Second)
+					timeout := r.SubscribeResolverTimeout
+					if timeout == 0 {
+						timeout = time.Second
+					}
```

Comment:
I like this one and I'd be happy to merge it. Not sure how I've missed the hard-coded `time.Second`. I'll cherry pick it in a separate PR, though.

Reply:
Extracted into it's own PR #418